be/src/storage/primary_index.cpp

```diff
@@ -1414,4 +1415,40 @@ Status PrimaryIndex::major_compaction(Tablet* tablet) {
     }
 }
 
+Status PrimaryIndex::reset(Tablet* tablet, EditVersion version, PersistentIndexMetaPB* index_meta) {
+    _table_id = tablet->belonged_table_id();
+    _tablet_id = tablet->tablet_id();
+    const TabletSchemaCSPtr tablet_schema_ptr = tablet->tablet_schema();
+    vector<ColumnId> pk_columns(tablet_schema_ptr->num_key_columns());
+    for (auto i = 0; i < tablet_schema_ptr->num_key_columns(); i++) {
+        pk_columns[i] = (ColumnId)i;
+    }
+    auto pkey_schema = ChunkHelper::convert_schema(tablet_schema_ptr, pk_columns);
+    _set_schema(pkey_schema);
+
+    size_t fix_size = PrimaryKeyEncoder::get_encoded_fixed_size(_pk_schema);
+
+    if (tablet->get_enable_persistent_index() && (fix_size <= 128)) {
```

Comment:
`fix_size <= 128`  this will trigger full memory index?

Reply:
> `fix_size <= 128` this will trigger full memory index?

From the code logic does look like this, there is no change in the judgment logic here, to keep the same logic as before.